glide.yaml

```diff
@@ -22,3 +20,11 @@ import:
   version: master
 - package: github.com/giantswarm/versionbundle
   version: master
+
+
+- package: k8s.io/apiextensions-apiserver
+  version: kubernetes-1.8.5
+- package: k8s.io/apimachinery
+  version: kubernetes-1.8.5
+- package: k8s.io/client-go
+  version: kubernetes-1.8.5
```

Comment:
These have to be pinned because k8scloudconfig still refers to older packages. So we overwrite it here, otherwise glide does not know how to do it right.

Reply:
Why k8scloudconfig refers to k8s.io packages?

Reply:
I actually don't know. It should not, right? I just pinned this here to get it done.

Reply:
Oh, right now it imports apiextensions. But that's not the problem. I think you had to pinned that because of dep. When you switch to dep you probably can remove those.